crates/sui-bridge/src/server/mod.rs

```diff
@@ -38,6 +39,7 @@ pub(crate) mod mock_handler;
 pub const APPLICATION_JSON: &str = "application/json";
 
 pub const PING_PATH: &str = "/ping";
+pub const NETWORK_KEY_PATH: &str = "/metrics/network_key";
```

Comment:
do we need this when we can get `BridgeNodePublicMetadata` from the `ping` endpoint ?

Reply:
Oh my bad, I called this out in the PR description:

> I'm also adding a new api endpoint, /metrics/network_key which returns only the network key, this isn't necessary, as /ping already returns this info, I'm open to removing it.

I don't feel strongly, just thought it might be bad practice to rely on `/ping`?

Reply:
could you elaborate?

Reply:
In my experience,`/ping` is generally used as a healthcheck, for example if running in k8s it's the thing that indicates if a pod is "ready" or not by simply returning a `200` status. If I were making changes to the bridge, I wouldn't expect there to be apps dependent on the response contents of the `/ping` endpoint